packages/back-end/generated/spec.yaml

```diff
@@ -205,6 +420,62 @@ paths:
                 properties:
                   feature:
                     $ref: '#/components/schemas/Feature'
+    post:
```

Comment:
Should this be a PUT ? or a PATCH if partial updates are allowed

Reply:
We are establishing a convention to use `POST` for partial updates for simplicity